### PR TITLE
Build scenario

### PR DESCRIPTION
**Pull request by sumerc**

### CODE REVIEW

File: .blackfire.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    #!blackfire-player
+
+    scenario
+        name "Basic Visit"
+        visit url("/")
+            name "Homepage"
+            expect status_code() == 200
+            expect css("tbody.js-sightings-list tr").count() > 10
+
+        #click css('tbody.js-sightings-list tr a')
```
Comment (lolautruche) [inline .blackfire.yml]: You can probably remove this line

File: .platform.app.yaml [hunk 2]
```diff
@@ -74,6 +74,7 @@ hooks:
         rm blackfire.tar.gz blackfire blackfire.sha1
     )
 
+    pip install blackfire-1.5.5_internal.14-cp39-cp39-manylinux2010_x86_64.whl
```
Comment (lolautruche) [inline .platform.app.yaml]: I guess we'll remove this line once the new version is out?

File: README.md [hunk 3]
```diff
@@ -29,3 +29,5 @@ Tested only with Python 3.9.
    ```bash
    blackfire-python manage.py runserver
    ```
+
+test
```
Comment (lolautruche) [inline README.md]: To be removed

**Review by lolautruche — CHANGES_REQUESTED** [review]

File: .platform.app.yaml (hunk 2 above)
Comment (sumerc) [reply .platform.app.yaml]: Yes

File: README.md (hunk 3 above)
Comment (sumerc) [reply README.md]: Yes!

**Review by lolautruche — APPROVED** [review]

**Pull request merged**